tests/upgrade/postgres_run.sh

```diff
@@ -213,6 +213,8 @@ test_upgrade_paths() {
     info "Rolling back to previous version with Postgres still enabled"
     force_rollback_to_previous_postgres
     wait_for_api
+    # Pulls can take a long time so bounce Sensor to cancel out any restarts
+    kubectl -n stackrox delete po -l app=sensor --grace-period=0
```

Comment:
Should we do a `sensor-wait`?
Should we bounce collector too?

Reply:
Yeah I guess we should do a sensor-wait and a collector bounce. Perhaps it'd be better to exclude a restart for this case

Reply:
Yeah if it is a restart you are trying to fix, adding it to `restart-ok-patterns.json` probably makes sense.  We already have a similar one for this test where collector bounces because it can't download the kernels.